Improve error message when using -g option with links instead of links2.
(Closes: #602227)

Index: links2/main.c
===================================================================
--- links2.orig/main.c	2012-06-26 14:17:54.000000000 +0200
+++ links2/main.c	2012-06-26 14:41:34.000000000 +0200
@@ -355,7 +355,7 @@
 			init_dither(drv->depth);
 			F = 1;
 #else
-			fprintf(stderr, "Graphics not enabled when compiling\n");
+			fprintf(stderr, "Graphics not enabled when compiling (use links2 instead for graphics mode)\n");
 			retval = RET_SYNTAX;
 			goto ttt;
 #endif
